docs/extend/authorization.md

```diff
@@ -0,0 +1,120 @@
+<!--[metadata]>
++++
+title = "Access authorization plugin"
+description = "How to create authorization plugins to manage access control to your Docker daemon."
+keywords = ["security, authorization, authentication, docker, documentation, plugin, extend"]
+[menu.main]
+parent = "mn_extend"
+weight = -1
++++
+<![end-metadata]-->
+
+
+# Access authorization
+
+By default Docker’s authorization model is all or nothing. Anyone who has access to the Docker daemon has the ability to run any Docker command. Authorization plugins enable creating granular access policies for managing access to Docker daemon. 
+
+## Basic principles
+
+* The authorization sub-system is based on Docker's [Plugin API](http://docs.docker.com/engine/extend/plugin_api), so that anyone could add / develop an authorization plug-in. It is possible to add a plug-in to a deployed Docker daemon without the need to rebuild the daemon. Daemon restart is required.
+
+* The authorization sub-system enables approving or denying requests to the Docker daemon based on the current user and command context, where the authentication context contains all user details and the authentication method, and the command context contains all the relevant request data.
```

Comment:
```
The authorization sub-system enables approving or denying requests,
based on the "authentication" and "command" context provided in the
request. The "authentication" context contains all user details, and
the "command" context contains all relevant request data.
```

Trying to avoid "current user" here, because afaik, the API is stateless, so there's no "current" user,
only the information that is provided in the request (?)